app/views/spree/layouts/admin.html.haml

```diff
@@ -1,85 +1,84 @@
+!!!
```

Comment:
what does this do?

Reply:
it adds the doctype tag in the very beginning of each page.
which basically means "this doc is html5"
![Screenshot 2019-07-29 at 17 00 58](https://user-images.githubusercontent.com/1640378/62063280-7995bf80-b222-11e9-8338-138978ca9b37.png)

Reply:
otherwise it defaults to html4/legacy